### DISCUSSION TITLE
Windows 11 widgets reposition

### DISCUSSION THREAD

**Discussion by Diego-JRH** [valinet/ExplorerPatcher#3421] [Ideas]
In the new windows 11 release the widgets icon moved to the right when taskbar is left-aligned. It also grew in size. I would like to be able to position it where it was before and to be the size of a standard taskbar icon.
I hope I explained it clearly.

**Comment by bbmaster123** [comment]
set 48660958 (ExtendedModeAEPForTaskbar) to disabled with vivetool, and restart explorer :)
